authentication/KratosApiAuthentication.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * This script is used to authenticate a user using Ory Kratos self-service API for clients without browsers.
+ *
+ * Authentication Verification can be configured with the following rule:
+ * - Verification Strategy: Poll the Specified URL
+ * - Regex Pattern used to identify Logged In message: \Qactive.*
```

Comment:
This pattern does not look right, missing `\E` before `.*`? Or remove `\Q`? (Some of the comments added in this script also apply to the other.)

Reply:
Oh yes, thanks for pointing this out! That makes the tests much faster ;)